File: aoj/volume100/python3/A10026.py
while True:
    no_items = int(input())
    if no_items == 0:
        break

    nums = list(map(int, input().split()))

    average = sum(nums) / len(nums)
    variance = sum([(num-average) ** 2 for num in nums]) / len(nums)
    standard_deviation = variance ** 0.5

    print('%.6f' % standard_deviation)

File: aoj/volume100/nodejs/A10026.js
function main(lines) {
  for (var i = 0;;) {
    var noItems = parseInt(lines[i++], 10);
    if (noItems === 0)
      break;
 
    var items    = lines[i++].split(/\s+/).map(parseFloat);
    var average  = items.reduce(function(sum, n){ return sum + n }, 0) / items.length;
    var variance = items.reduce(function(sum, n){ return sum + Math.pow(n-average, 2)}, 0) / items.length;
    var standarddeviation = Math.sqrt(variance);
    console.log(standarddeviation.toFixed(6));
  }
}
 
var chunks = [];
process.stdin.on('data', function(chunk) {
  chunks.push(chunk)
});
process.stdin.on('end', function() {
  var lines = chunks.join('').split('\n');
  main(lines);
});
